stackslib/src/chainstate/stacks/transaction.rs

```diff
@@ -29,6 +29,7 @@ use stacks_common::types::StacksPublicKeyBuffer;
 use stacks_common::util::hash::{to_hex, MerkleHashFunc, MerkleTree, Sha512Trunc256Sum};
 use stacks_common::util::retry::BoundReader;
 use stacks_common::util::secp256k1::MessageSignature;
+use wsts::curve as p256k1;
```

Comment:
This statement will likely make the diff rather smaller, but ideally we'd go ahead and remove any mention of `p256k1`.

Not worth holding this up, but eventually it would be good to fix this

Reply:
Moved the `use` statements to top-level and removed the `p256k1` alias